docs/UserGuide.md

```diff
@@ -96,15 +96,25 @@ Examples:
 
 ### Listing all persons : `list`
 
-Shows a list of all persons in the address book, automatically sorted by their priority from HIGH to LOW.
+Shows a list of persons in the address book, automatically sorted by their priority from HIGH to LOW.
```

Comment:
Could you rewrite the description as the sorting isn't preserved anymore?

Reply:
Can you clarify what you mean by preserved? The sorting order is still preserved after the user has run the sort command.
See https://github.com/AY2425S1-CS2103T-F14b-1/tp/pull/159#discussion_r1832004784

Reply:
As mentioned by @arjun2598:

> Perhaps this behaviour could be explained here so the user is not misled to believe that **listing always sorts in priority order**?